system/host_cpu.tf

```diff
@@ -0,0 +1,47 @@
+resource "datadog_monitor" "cpu_average" {
+  count   = "${var.monitor_enabled}"
+  name    = "${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High CPU usage last ${var.cpu_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
+  query   = "avg(last_${var.cpu_time}):avg:system.cpu.system{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.cpu_critical_threshold_value}"
```

Comment:
I'm not keen on hardcoding the selection criteria:`host:${data.aws_instance.monitored.instance_id}` How would we monitor an Auto Scaling Group?

Reply:
@osterman , then we should think about case like bare-metal or VPS or or or or... and it'll never finished. We should have some stable version and implement additional features one by one in next versions.

Reply:
We should close this PR and open a smaller PR. One for each monitor.